documentation/src/main/markdown/runningdetect/autonomousscan.dita

```diff
@@ -0,0 +1,344 @@
+                            <entry>Online</entry>
+                            <entry>STATELESS</entry>
+                        </row>
+                        <row>
+                            <entry/>
+                            <entry/>
+                            <entry/>
+                            <entry>STATELESS</entry>
+                            <entry/>
+                            <entry>Online</entry>
+                            <entry>STATELESS</entry>
+                        </row>
+                        <row>
+                            <entry/>
+                            <entry>configured</entry>
+                            <entry>false</entry>
+                            <entry/>
+                            <entry>any value</entry>
+                            <entry>Online</entry>
+                            <entry>INTELLIGENT</entry>
+                        </row>
+                        <row>
+                            <entry/>
+                            <entry/>
+                            <entry/>
+                            <entry>INTELLIGENT</entry>
+                            <entry/>
+                            <entry>Online</entry>
+                            <entry>INTELLIGENT</entry>
+                        </row>
+                    </tbody>
+                </tgroup>
+            </table>
+        </body>
+    </topic>
+    <topic id="requirements-and-limitations">
+        <title>Requirements and Limitations</title>
+        <body/>
+        <topic id="general-requirements">
+            <title>General Requirements</title>
+            <body>
+                <ul>
+                    <li>
+                        <p>Scans require local network connectivity when used with <keyword keyref="blackduck_product_name"/> or if the scan location is remote,
+                            remote network connectivity is required.</p>
+                    </li>
+                    <li>
+                        <p><keyword keyref="blackduck_product_name"/> must be configured for binary
+                            scans.</p>
+                    </li>
+                </ul>
+            </body>
+        </topic>
+        <topic id="limitations">
+            <title>Limitations</title>
+            <body>
+                <ul>
+                    <li>
+                        <p>Autonomous scanning does not support flags.</p>
+                    </li>
+                    <li>
+                        <p><keyword keyref="blackduck_product_name"/> Snippet scans are not
+                            supported.</p>
+                    </li>
+                    <li>
+                        <p>The following settings will not be used or persisted by <keyword keyref="company_name"/>
+                            <keyword keyref="solution_name"/> when running in Autonomous mode:</p>
+                        <ul>
+                            <li>
+                                <p>--detect.tools</p>
```

Comment:
--detect.tools ?
@gopannabd we are using and persisting it now.

Reply:
@sig-cpotts this would be incorrect, we have changed it now. Please change docs and add that all the properties except these paths will be supported:
DETECT_OUTPUT_PATH
DETECT_BDIO_OUTPUT_PATH
DETECT_SCAN_OUTPUT_PATH
DETECT_TOOLS_OUTPUT_PATH
DETECT_IMPACT_ANALYSIS_OUTPUT_PATH
DETECT_STATUS_JSON_OUTPUT_PATH

Reply:
@devmehtasynopsys changes made. Thank you.